### ISSUE TITLE
Drop CoffeeScript in favor of ES6 (Babel)

### ISSUE THREAD

**Issue by silvenon** [yeoman/generator-chromeapp#57]
Ref. https://github.com/yeoman/generator-webapp/pull/543#issuecomment-111724489.

**Comment by arthurvr** [comment]
Yes please!

**Comment by sindresorhus** [comment]
:+1:

**Comment by arthurvr** [comment]
Happy to work on this. Optionally or do with ES2015ify all the things?

**Comment by silvenon** [comment]
Can you rephrase the question?

**Comment by arthurvr** [comment]
Do we make babel an option or just the default?

**Comment by silvenon** [comment]
Default. It's just JavaScript anyway, I don't think anyone can possibly mind.

**Comment by ragingwind** [comment]
:+1:
